README.md

```diff
@@ -58,13 +58,22 @@ To start the app, run these commands, replacing `$(pwd)/workspace` with the dire
 ```bash
 # The directory you want OpenDevin to work with. MUST be an absolute path!
 export WORKSPACE_BASE=$(pwd)/workspace;
+
+# Your OpenAI API key, or any other LLM API key
+#export LLM_API_KEY="sk-..."
+export LLM_API_KEY="sk-"
+export LLM_MODEL="groq/llama3-70b-8192"
```

Comment:
Sugesst default to gpt 4o
```suggestion
export LLM_MODEL="azure/gpt-4o"
```

Reply:
These should not  be in the README, they are set via the UI